cores/nRF5/sysview/SEGGER/SEGGER_RTT.c

```diff
@@ -52,15 +42,16 @@
 *                                                                    *
 **********************************************************************
 *                                                                    *
-*       SystemView version: V2.52d                                    *
+*       RTT version: 6.60d                                           *

```

Comment:
This is not important but you seem to use the rtt file bundled with jlink. Since jlink version change very often, I much prefer the **Target Sources** download, since it is tested against the SystemView in specific version.

https://www.segger.com/downloads/systemview/